src/docs/resources/libraries/README.md

```diff
@@ -28,6 +28,7 @@ The below table lists the different .NET libraries used in Orchard Core:
 | [Newtonsoft.Json](https://github.com/JamesNK/Newtonsoft.Json) | Json.NET is a popular high-performance JSON framework for .NET | 13.0.1 | [MIT](https://github.com/JamesNK/Newtonsoft.Json/blob/master/LICENSE.md) |
 | [Noda Time](https://github.com/nodatime/nodatime) | A better date and time API for .NET. | 3.0.9 | [Apache-2.0](https://github.com/nodatime/nodatime/blob/master/LICENSE.txt) |
 | [OpenIddict](https://github.com/openiddict/openiddict-core) | Versatile OpenID Connect stack for ASP.NET Core. | 3.1.1 | [Apache-2.0](https://github.com/openiddict/openiddict-core/blob/dev/LICENSE.md)) |
+| [PdfPig](https://github.com/UglyToad/PdfPig/) | Library to read and extract text and other content from PDF files. | 0.1.5-alpha001 | [Apache-2.0](https://github.com/UglyToad/PdfPig/blob/master/LICENSE) |
```

Comment:
It's no longer alpha https://www.nuget.org/packages/PdfPig/

Reply:
Indeed, thanks, updated.